ship/src/main/scala/ch/epfl/bluebrain/nexus/ship/EventProcessor.scala

```diff
@@ -31,27 +30,39 @@ object EventProcessor {
 
   private val logger = Logger[EventProcessor.type]
 
-  def run(eventStream: Stream[IO, InputEvent], processors: EventProcessor[_]*): IO[Offset] = {
+  def run(eventStream: Stream[IO, InputEvent], processors: EventProcessor[_]*): IO[ImportReport] = {
     val processorsMap = processors.foldLeft(Map.empty[EntityType, EventProcessor[_]]) { (acc, processor) =>
       acc + (processor.resourceType -> processor)
     }
     eventStream
-      .evalTap { event =>
+      .evalScan(ImportReport.start) { case(report, event) =>
         processorsMap.get(event.`type`) match {
           case Some(processor) =>
-            processor.evaluate(event).onError { err =>
+            processor.evaluate(event)
+              .map { status =>
```

Comment:
Not sure if it'll be useful to preserve, but could also do `.attempt.map`, log the errors and include a failure case with the error info, as well as dropped / success. I've done migrations like this and something like this can help to categorise classes of errors to know where to focus on

Reply:
This PR is meant to be just a first step meant to be enriched later when we import more cases and have a better grasp on how to handle error handling / error handling (with something similar to what you propose for example).